config/store.go

```diff
@@ -60,3 +61,47 @@ func NewStore(dsn string, watch bool) (Store, error) {
 
 	return NewFileStore(dsn, watch)
 }
+
+func MigrateConfig(sourceStore, destinationStore Store) error {
+	// Copy config from source to destination
+	sourceStoreConfig := sourceStore.Get()
+
+	_, err := destinationStore.Set(sourceStoreConfig)
+	if err != nil {
+		return errors.Wrap(err, "failed to migrate config")
+	}
+
+	//get the certificate files from sourceStore
```

Comment:
nit: Can you make the comments uniform with others in this file?
```suggestion
	// Get the certificate files from sourceStore
```